client/src/components/ReviewForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import { useLocation } from "react-router-dom"
+
+import translateServerErrors from "../services/translateServerErrors.js"
+
+const ReviewForm = (props) => {
```

Comment:
the feature of this branch is for adding a new site - logic for adding a new review needs be a separate feature (unless this is brought in from a merge, however I do not see this ReviewForm in the `main` branch. Let's remove and tackle in it's own feature

Reply:
I noticed yesterday when I made a new branch off of that my starting code had this (beginning of) a component, even though we hadn't merged it to main - I think I must have done so locally on my machine, so now when I branch out from main (at its current state on my machine) it has this. So this is on me!